rabbitmq-utils/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM rabbitmq:3
+
+RUN apt-get update && apt-get install -y iputils-ping telnet curl python
+COPY files/rabbitio-v0.5.4-linux-amd64 /opt/rabbitio
```

Comment:
I'd rather download it from https://github.com/meltwater/rabbitio/release instead of copying it into our repo

Reply:
Done